.kodiak/config.yaml

```diff
@@ -0,0 +1,30 @@
+version: 1.0
+
+snow:
+  - id: 545562 # Milo CC
+
+notifications:
+  jira:
+    default:
+      project: MWPW # Mandatory
+      filters:
+        include:
+          risk_rating: R5
+      fields:
+        assignee:
+          name: seanchoi
```

Comment:
Should all milo cc tickets be assigned to @seanchoi-dev automatically? Not sure this scales over time.

Reply:
+1. Could we get members from CC team assigned to the tickets?
@salonijain3

Reply:
@honstar Unfortuantely Kodiak does not currently allow assigning to a group, but Sean can delegate as needed. If there's a different individual who is better suited to do this, I'm happy to assign them.

@aishwaryamathuria Can add anyone you'd like, just need to know who else should be watchers on the ticket.

Reply:
@thedoc31 Can you please add @salonijain3 and myself as watchers? One of us get make sure that these tickets get assigned.

Reply:
Also, @thedoc31 you can assign all tickets to <suman h> ldap : <sumanh@adobe.com> for adobecom/cc and then we can reassign accordingly.

Reply:
Alright, changes are committed @dstrong23 @salonijain3